<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QMainWindow" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="maximumSize">
        <size>
         <width>193</width>
         <height>341</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="SwitchToContactsMode">
          <property name="maximumSize">
           <size>
            <width>191</width>
            <height>22</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>All Contacts</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>My Contacts</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listContacts">
          <property name="maximumSize">
           <size>
            <width>191</width>
            <height>311</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Connect">
          <property name="text">
           <string>Connenct</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddContact">
          <property name="text">
           <string>Add contact</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="Chat">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>250</height>
         </size>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="InputChat">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="Send">
             <property name="text">
              <string>Send</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Attach">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Attach</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
